src/algorithms/tracking/CKFTracking.cc

```diff
@@ -10,6 +10,7 @@
 #include <Acts/EventData/GenericBoundTrackParameters.hpp>
 #include <Acts/EventData/TrackStateProxy.hpp>
 #include <Acts/EventData/Types.hpp>
+#include <sys/types.h>
```

Comment:
This is not portable. Should not go in, in my opinion.

Reply:
This was suggested because of use of ssize_t, which is not a C++ type. Removed in 8b7d37b.